packages/node_modules/nav-frontend-skjema/md/fnrinput/FnrInput.overview.mdx

```diff
@@ -0,0 +1,23 @@
+import Example from './../../../../../guideline-app/app/ui/components/example/Example';
+import Alertstripe from 'nav-frontend-alertstriper';
+import { FnrInput } from './../../';
+
+## Normal
+
+<Example>
```

Comment:
Hadde det ikke vært fint om dette eksempelet faktisk kunne demonstrere denne valideringen med noe lokal state som toggler noe ala `feil={ (this.state.invalid) ? {feilmelding: 'Ugyldig fødselsnummer'} : undefined }` her?

Se gjerne på [dokumentasjonen for Modal](https://github.com/navikt/nav-frontend-moduler/tree/master/packages/node_modules/nav-frontend-modal) for hvordan du kan skille ut disse eksemplene i egne filer, og bygge en liten mini-app her.

Reply:
done